cfep-09.md

```diff
@@ -0,0 +1,261 @@
+
+<table>
+<tr><td> Pinning Proposal System for Automatic Rebuilds</td><td> Pinning Proposal </td>
+<tr><td> Status </td><td> Proposed </td></tr>
+<tr><td> Author(s) </td><td> Christopher J. "CJ" Wright &lt;cjwright4242@gmail.com&gt;; Matthew R. Becker &lt;becker.mr@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> Feb 20, 2019</td></tr>
+<tr><td> Updated </td><td> Aug 30, 2016</td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+This CFEP proposes a change to how our internal pinning system works.
+The new system will comprise of a new repo for proposed migrations.
```

Comment:
I think the abstract needs adjustment since we will use a migrations folder in the pinnings feedstock.

Reply:
This comment has not been addressed.

Reply:
Sorry about that, I think I have fix this now